src/access/ownable/dual_ownable.cairo

```diff
@@ -1,21 +1,15 @@
 // SPDX-License-Identifier: MIT
 // OpenZeppelin Contracts for Cairo v0.7.0 (access/ownable/dual_ownable.cairo)
 
-use array::ArrayTrait;
-use array::SpanTrait;
-use core::result::ResultTrait;
```

Comment:
how did you identify them? language server?

Reply:
Manually (after reading a post in the forum saying that most of the corelib Cairo-related improts (not starknet) were not necessary anymore). I'm sure there are more, and at some point the language server should include that feature, but no yet sadly :(